models/app.models.js

```diff
@@ -21,4 +21,14 @@ exports.fetchArticleById = async id => {
 	} else {
 		return results.rows[0];
 	}
-};
\ No newline at end of file
+};
+
+exports.fetchArticles = async () => {
+	const results = await db.query('SELECT * FROM articles');
+
+	if (results.rowCount === 0) {
+		return Promise.reject({ code: 404 });
+	} else {
```

Comment:
very good model! I do believe it's missing some functionality from the ticket however